### DISCUSSION TITLE
Obsidian crash - "Segmentation fault (core dump)"

### DISCUSSION THREAD

**Discussion by Graphiel** [hyprwm/Hyprland#8215] [Q&A] [answered]
I already tried to run _Obsidian_ with `obsidian --disable-gpu` and on many other ways with many other flags (included the ones who were at the _Hyprland's Wiki_). However, none of them works.

What happens is:
1. Run _Obsidian_, both with or without _flags_.
2. After some 2~3 seconds it just crashes.

More strangely, i got this output:

**$ obsidian --disable-gpu**

`obsidian --disable-gpu`
`2024-10-24 01:23:20 Loading main app package /usr/lib/obsidian/obsidian.asar`
`Ignored: Error: ENOENT: no such file or directory, open '/home/graphiel/.config/obsidian/obsidian.json'`
`MESA-INTEL: warning: Ivy Bridge Vulkan support is incomplete`
`2024-10-24 01:23:21 Checking for update using Github`
`2024-10-24 01:23:21 Success.`
`2024-10-24 01:23:21 Latest version is 1.7.4`
`2024-10-24 01:23:21 App is up to date.`
`Segmentation fault (core dump)`

Also, `env = ELECTRON_OZONE_PLATFORM_HINT , auto` is already on hyprland.conf

> Note: Never got _Obsidian_ running on this setup, i just installed _Hyprland_ after a fresh _Arch_ install (with some tweaks, of course).
Got through some forums and searched a bit, while i could find some useful similar info i couldn't solve my problem.

Edit:
Just adding more useful info.

**$ obsidian**
`[...]`
`ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!`
`Segmentation fault (core dump)`

**Comment by Graphiel** [comment, marked as answer]
**[SOLVED!!!]**
If you have the same problem as my own, well... maybe the _cursor_ is the one causing the trouble.
_Yes, the _cursor_._

**[Potential Fix]**
Find a way to change the cursor on your _DE/WM_ and just change to a random one just for testing purposes.
I used [_GTK Settings_ (i.e., _nwg-look_)](https://github.com/nwg-piotr/nwg-look).

**Reply by lstuma** [reply]
Yooo I can't believe this worked!!! Thank you :))

**Reply by craigccfl** [reply]
All day been tweeking, looking for the fix. Change the curser with nwg-look and by miracle, it worked.  This should be front page news because others are having the same errors.
